client/apps/edit/components/admin/appearances/index.coffee

```diff
@@ -0,0 +1,71 @@
+React = require 'react'
+ReactDOM = require 'react-dom'
+{ div, label, input } = React.DOM
+request = require 'superagent'
+sd = require('sharify').data
+
+AutocompleteSelect = require '../../../../../components/autocomplete_select/index.coffee'
+AutocompleteList = React.createFactory require '../components/autocomplete.coffee'
+
+module.exports = React.createClass
+  displayName: 'AdminAppearances'
+
+  componentDidMount: ->
+    @setupBiographyAutocomplete()
+    ReactDOM.findDOMNode(@refs.container).classList += ' active'
+
+  componentWillUnmount: ->
+    $(@refs.biography_for_artist_id).each (i, ref) ->
+      ReactDOM.unmountComponentAtNode(ref)
+
+  setupBiographyAutocomplete: ->
+    new AutocompleteSelect $(@refs.biography_for_artist_id)[0],
+      url: "#{sd.ARTSY_URL}/api/v1/match/artists?term=%QUERY"
+      placeholder: 'Search artist by name...'
+      filter: (artists) -> for artist in artists
+        { id: artist._id, value: artist.name }
+      selected: (e, item) =>
+        @props.onChange 'biography_for_artist_id', item.id
+      removed: =>
+        @props.onChange 'biography_for_artist_id', null
+      idToFetch: @props.article.get('biography_for_artist_id')
+      fetchUrl: sd.ARTSY_URL + '/api/v1/artist/' + @props.article.get('biography_for_artist_id')
+      resObject: (res) ->
+        value: res.body.name, id: res.body.id
```

Comment:
What's the reason for not including this in the render function? It _should_ work the same as AutocompleteList unless I forgot to add some functionality?